hardware/BleBox.cpp

```diff
@@ -168,7 +200,7 @@ std::string BleBox::IPToHex(const std::string &IPAddress)
 	std::vector<std::string> strarray;
 	boost::split(strarray, IPAddress, boost::is_any_of("."));
 	if (strarray.size() != 4)
-		return "0.0.0.0";
+		return "";
```

Comment:
I didn't see StringSplit (i changed to this method)
I input ip adress but result is hex for example "0C1B2E98". I save it on this form in database, because later i must get this ip, when handle WriteToHardware - and decode it in this method std::string BleBox::GetDeviceIP(const tRBUF *id)